local/flake.nix

```diff
@@ -0,0 +1,51 @@
+{
+  description = "Local Development Environment";
+  inputs.nosys.url = "github:divnix/nosys";
+  inputs.nixpkgs.url = "github:nixos/nixpkgs/nixpkgs-unstable";
+  inputs.devshell.url = "github:numtide/devshell";
+  inputs.call-flake.url = "github:divnix/call-flake";
+  outputs = inputs @ {
+    nosys,
+    call-flake,
+    ...
+  }:
+    nosys ((call-flake ../.).inputs // inputs) (
+      {
+        self,
+        nixpkgs,
+        devshell,
+        std,
+        ...
+      }:
+        with nixpkgs.legacyPackages;
+        with nixpkgs.legacyPackages.nodePackages;
+        with devshell.legacyPackages; let
+          inherit (lib.stringsWithDeps) noDepEntry;
+          checkMod = {
+            commands = [{package = treefmt;}];
+            packages = [
+              alejandra
+              shfmt
+              prettier
+              prettier-plugin-toml
+            ];
+            devshell.startup.nodejs-setuphook = noDepEntry ''
+              export NODE_PATH=${prettier-plugin-toml}/lib/node_modules:$NODE_PATH
```

Comment:
```suggestion
              export NODE_PATH=${prettier-plugin-toml}/lib/node_modules:$NODE_PATH
              export PATH=$PATH:$PRJ_ROOT/node_modules/.bin
```

Reply:
If I understand this suggestion properly, this would move the responsibility to procure the prettier-plugin-toml binary to `node_modules` folder and I don't know if it's there, but even if it is "outsourcing" that contract (without actually having established one) seems a bit wonky in terms of maintenance.

Reply:
The idea here was to make the better developer experience, we can 'outsource' it or have 2 of them installed

Reply:
Got ya! I'd say let's do a closer integration in a next step, once this proposal has landed and people are a bit more familiar with "what this is".